docs/advanced/addons/pcidevices.md

```diff
@@ -10,6 +10,8 @@ title: "PCI Devices"
 
 _Available as of v1.1.0_
 
+_Unavailable on ARM64 architecture_
```

Comment:
```suggestion
_Currently incompatible with ARM64_
```

Reply:
We already have content like `_Available as of v1.2.0_`, I would prefer `_Unavailable on ARM64 architecture_`; maybe put them in  same line is better.

Reply:
Availability and compatibility are two different concepts. You can put them in the same line if you want, but I do not agree with using the draft wording.